go.mod

```diff
@@ -4,7 +4,8 @@ go 1.21.12
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.12.1
-	github.com/ava-labs/avalanchego v1.11.11-0.20240729205337-a0f7e422bb84
+	github.com/antithesishq/antithesis-sdk-go v0.3.8
```

Comment:
this seems to add some dependencies, not sure if the binary size matters.
perhaps it can be its own binary?

Reply:
I was wondering if it is cleaner to make a separate go.mod for the antithesis binary, but I am not sure if that adds too much complexity for what it brings.